src/main/java/no/unit/marc/RecordParser.java

```diff
@@ -97,7 +93,7 @@ public Reference parse(String xml) throws IOException,
         return reference;
```

Comment:
public Reference parse(String xml .. kaster checked exceptions som ikke ser ut til å være nødvendige

Reply:
You are right. De har jeg slettet.